backend/app/member_handlers.go

```
@@ -0,0 +1,102 @@
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("error binding json data"))
+	}
+	userID, exists := c.Get("user_id")
+	if !exists {
+		return nil, UnAuthorized(errors.New("authentication is required"))
+	}
+	member := models.Member{ProjectID: memberInput.ProjectID, Admin: memberInput.Admin, UserID: userID.(string)}
+	if err := member.Validate(); err != nil {
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("input data is invalid"))
+	}
+
+	if fmt.Sprint(userID) != fmt.Sprint(member.UserID) && !member.Admin {
+		return nil, Forbidden(errors.New("you have no access to create member"))
+	}
+
+	addedMember, err := a.DB.CreateNewMember(member)
+	if err == gorm.ErrDuplicatedKey{
+		log.Error().Err(err).Send()
+		return nil, Forbidden(errors.New("member already exists"))
+	}
+	if err != nil {
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("error creating new member"))
+	}
+
+	return ResponseMsg{
+		Message: "member created successfully",
+		Data:    addedMember,
+	}, Created()
+}
+
+func (a *App) getAllMembers(c *gin.Context) (interface{}, Response) {
+	members, err := a.DB.GetAllMembers()
+	if err != nil {
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("error getting all members"))
+	}
+
+	return ResponseMsg{
+		Message: "all members retrieved successfully",
+		Data:    members,
+	}, Ok()
+}
+
+func (a *App) updateMemberOwnership(c *gin.Context) (interface{}, Response) {
+	memberInput := createMemberInput{}
+
+	if err := c.ShouldBindJSON(&memberInput); err != nil {
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("error binding json data"))
+	}
+	userID, exists := c.Get("user_id")
+	if !exists {
+		return nil, UnAuthorized(errors.New("authentication is required"))
+	}
+
+	id, err := strconv.Atoi(c.Param("id"))
+	if err != nil {
+		log.Error().Err(err).Send()
+		return nil, InternalServerError(errors.New("error parsing id"))
```

please return the constant error msg for all `InternalServerError`